pkg/csi_driver/gcfs_driver.go

```diff
@@ -328,6 +329,10 @@ func (driver *GCFSDriver) Run(endpoint string) {
 	// Start the nonblocking GRPC.
 	s := NewNonBlockingGRPCServer()
 	s.Start(endpoint, driver.ids, driver.cs, driver.ns)
+	if driver.config.RunNode && driver.config.FeatureOptions.FeatureLockRelease.Enabled && !driver.config.FeatureOptions.FeatureLockRelease.V2 {
```

Comment:
I don't think `v2` is a suitable name, as it might be too vague for others who lack context about the lock release functionality. How about naming the flag `builtin-lock-release-controller`? We could then structure the conditional like this:

```go
if driver.config.RunNode && driver.config.FeatureOptions.FeatureLockRelease.Enabled && driver.config.FeatureOptions.FeatureLockRelease.builtinLockReleaseController {
    // Start the lock release controller on the node driver.
    driver.ns.(*nodeServer).lockReleaseController.Run(context.Background())
}
```

This makes the purpose of the flag clearer to anyone reading the code.

Reply:
+1 on a more intuitive name other than v2. But @tyuchn can you please elaborate your thought on what does "builtin" mean

Reply:
I meant that the lock release controller runs as part of the CSI driver. Perhaps builtin isn’t the best term, but my point is to use a more descriptive flag to control whether the lock release controller runs in standalone mode or as part of the CSI driver.

Reply:
how about we change the names to 

type FeatureLockRelease struct {
    builtIn bool # <description>
    standAlone bool # <description>
}

Reply:
I agree with the Standalone idea, but I will keep "Enabled". The flag is also used to make sure config maps are modified when volumes are staging/unstaging. This behavior does not change when we use the standalone controller. 

Think of the v1 feature flag as the light switch, and v2 feature flag as the switch of the light bulb's voice control function...